tests/python/tir-transform/test_tir_transform_simplify.py

```diff
@@ -1757,5 +1757,27 @@ def expected(a: T.handle):
         A[T.int64(1)] = T.float32(0)
 
 
+class TestNestedIfElimination(BaseBeforeAfter):
+    def before(a: T.Buffer((2, 8), "int32"), b: T.Buffer((2, 8), "int32")):
+        for i0, j0 in T.grid(2, 8):
+            with T.block("P"):
```

Comment:
Does the test case depend on the `BlockNode` being present?  If not, we should remove it.  (Removing the `with T.block`, `T.axis.remap`, `T.reads`, and `T.writes` lines, then using `i0` and `j0` directly.)

Reply:
Removed.